locales/en-US/strings.properties

```diff
@@ -110,7 +110,22 @@ time_label_minute={number}m
 time_label_hour={number}h
 time_label_day={number}d
 
-# LOCALIZATION NOTE (settings_pane_*): This is shown in the Settings Pane sidebar.
+# LOCALIZATION NOTE (prefs_*, settings_*): These are shown in about:preferences
+# for a "Firefox Home" section. "Home" should be localized matching the sidebar.
+prefs_home_header=Firefox Home Content
+prefs_home_description=Choose what content you want on your Firefox Home screen.
+prefs_restore_defaults_button=Restore Defaults
+# LOCALIZATION NOTE (prefs_section_rows_option): This is a semi-colon list of
+# plural forms used in a drop down of multiple row options (1 row, 2 rows).
+# See: http://developer.mozilla.org/en/docs/Localization_and_Plurals
+prefs_section_rows_option={num} row;{num} rows
+prefs_search_header=Web Search
+prefs_topsites_description=The sites you visit most
+prefs_topstories_description=High-quality content you might otherwise miss
+prefs_topstories_show_sponsored_label={provider} Sponsored Stories
```

Comment:
Add a note for `{provider}`? E.g. `{provider} is going to be replaced by the name of a new provider (e.g. 'Pocket')`

Reply:
Added